slamd/design_assistant/processing/llm_service.py

```diff
@@ -257,26 +264,17 @@ def _create_output_format_excerpt(cls):
 
 For you orientation, here are some examples (note that these are just examples and must not be included in the general knowledge provided; further make sure to use units that are conventional for the components you are proposing):
 
-Example 1:
-Water to Cement Ratio: 0.5
+Make sure to stick to exactly this format! Here are examples:
 
-Example 2:
-Fly Ash: 20%
+Example 1 Geopolymere Concrete:
 
-The full output format thus must look as follows:
+Weight of Powders: 300 kg, Water to Cement Ratio: 35 %,Weight of Additives 150 kg, Weight of Admixtures (superplasticizer) 15 kg, Composition of Powder Blend FA/GGBFS: 50/50, Composition of Liquid H2O/NaOH/Na2SiO3: 95/3/2 m%, Processing: ambient curing, Aggregates 1800 kg 
+                        
 
-NAME_OF_THE_COMPONENT 1: VALUE 1, NAME_OF_THE_COMPONENT 2: VALUE 2, NAME_OF_THE_COMPONENT 3: VALUE 3
+Example 2 OPC based Concrete:
 
-and so on until all components are included. Make sure to stick to exactly this format! Here are examples:
+OPC: 400 kg, Water to Cement Ratio: 42 %, Weight of Admixtures (superplasticizer) 18 kg, Recycled Aggregates: 10%, Aggregates 1780 kg
 
-Example 1:
-
-Fly Ash: 30%, GGBFS: 70%, Water to Cement Ratio: 0.4
-
-Example 2:
-
-Geopolymer: 30%, Water: 60%, Recycled Aggregates: 10%
-
-Example 3:
-Fly Ash: 40%, Water: 25%, Biochar: 10%, Rice Husk Ash: 10%, Recycled Glass Fines: 10%, Super Plasticizer: 5%
+Example 3 Alkali Activated Binder Composition :
+Fly Ash/GGBFS: 40/60, Water to Cement Ratio: 38 %, Biochar: 10%, Rice Husk Ash: 10%, Recycled Glass Fines: 10%, Super Plasticizer: 5%
```

Comment:
In dem File gibt es noch 2 print statements, die man entfernen kann

print(design_knowledge_prompt)

und 

print(prompt)